node/pkg/watchers/solana/client.go

```diff
@@ -772,9 +765,11 @@ func (s *SolanaWatcher) processTransaction(ctx context.Context, logger *zap.Logg
 			}
 		}
 	}
+
+	return
 }
 
-func (s *SolanaWatcher) processInstruction(ctx context.Context, logger *zap.Logger, slot uint64, inst solana.CompiledInstruction, programIndex uint16, tx *solana.Transaction, signature solana.Signature, idx int, isReobservation bool) (bool, error) {
+func (s *SolanaWatcher) processInstruction(ctx context.Context, logger *zap.Logger, rpcClient *rpc.Client, slot uint64, inst solana.CompiledInstruction, programIndex uint16, tx *solana.Transaction, signature solana.Signature, idx int, isReobservation bool) (bool, error) {
```

Comment:
Same as above - would it make sense to access `s.rpcClient` here instead? (And `s.logger` now that it exists)

Reply:
I confirmed that I do need to pass in `rpcClient`, but not `logger`. I made that change.